theme_vehicle/views/customizations.xml

```diff
@@ -348,25 +348,11 @@
     </xpath>
 </template>
 
-<!-- ======== IMAGE WALL ======== -->
-<template id="s_images_wall" inherit_id="website.s_images_wall">
-    <xpath expr="//img" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[2]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[3]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[4]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[5]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
-    </xpath>
-    <xpath expr="(//img)[6]" position="attributes">
-        <attribute name="class" remove="rounded" separator=" "/>
```

Comment:
@anso-odoo I don't see this in the task but I suppose that was discussed.

Reply:
Ooops, no, I think it's a conflict fix issue. Will fix this 👍

Reply:
@anso-odoo Ok then :)

Reply:
fixed ✅